Jenkinsfile

```diff
@@ -596,6 +598,19 @@ void mergePR(){
     }
 }
 
+def isChangeInUI(){
```

Comment:
We don't need a new method to check the diffs.
We can use Jenkins OOTB functions something similar to check if the changes are in UI folder
`when {
                            changeset '**/UI/**'
                   }`

Reply:
I tested using `when { changeset '**/UI/**' }` and it doesn't seem like is doing what we need. 

And from Jenkins doc the  changeset parameter: `Execute the stage if the build’s SCM changeset contains one or more files matching the given pattern ` It is not clear to me what SCM changeset, but seem like is not changes of underline PR.

Reply:
@yafeldman It is working fine in other pipelines. Please let me know the scenario in which it did not work so I can test it on other pipelines.

SCM changeset is the set of files that changed in a commit (in our case its a PR).

Reply:
I just added fake ui change and added `changeset "**/ui/**"` and run build: https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-5709/16/pipeline/87

and it still skips cypress stage